src/main.js

```diff
@@ -23,13 +27,14 @@ function getRandomIndex(array) {
 }
 
 function randomBook() {
-  var randomCoverImage = covers[getRandomIndex(covers)];
-  var randomTitle = titles[getRandomIndex(titles)];
-  var randomDesc1 = descriptors[getRandomIndex(descriptors)];
-  var randomDesc2 = descriptors[getRandomIndex(descriptors)];
-
-  updateCoverImg.src = randomCoverImage;
-  updateTitle.innerText = randomTitle;
-  updateDescriptor1.innerText = randomDesc1;
-  updateDescriptor2.innerText = randomDesc2;
+  coverImg.src = covers[getRandomIndex(covers)];
```

Comment:
Terrific restructuring here. This seems to comply extremely well with the Iteration 1 instructions in that it should create a new cover object with a fresh id every time the show random cover button is clicked, along with when the page is loaded.